node/bin/src/revm_consistency_checker/mod.rs

```diff
@@ -0,0 +1,5 @@
+pub mod bytecode_hash;
+pub mod helpers;
+pub mod node;
+pub mod revm_state_db;
+pub mod storage_diff_comp;
```

Comment:
I'd move this entire module into its own crate. From what I have seen there is no reason for it to be in `bin`